doc/postprocess.py

```diff
@@ -34,12 +34,15 @@ def process_tex(lines):
     """
     new_lines = []
     for line in lines:
-        if (line.startswith(r'\section{numpy.')
-            or line.startswith(r'\subsection{numpy.')
-            or line.startswith(r'\subsubsection{numpy.')
-            or line.startswith(r'\paragraph{numpy.')
-            or line.startswith(r'\subparagraph{numpy.')
-            ):
+        if line.startswith(
+            (
+                "\\section{numpy.",
+                "\\subsection{numpy.",
+                "\\subsubsection{numpy.",
+                "\\paragraph{numpy.",
+                "\\subparagraph{numpy.",
+            )
+        ):
```

Comment:
I'm not completely happy with this style, but OK.

Reply:
I tried to mimic the initial style, but the linter complained or the result was not readable. This format is black / ruff style, pretty standard nowadays. Initially I  didn't like that style either, but it is readable and can be consistently applied in all contexts. I am happy to change to try again to adapt the initial style.

Reply:
Part of my problem was that I couldn't think of anything that was a big improvement. Your new fix at least clearly shows the condition of the `if`, which always helps when reading the code.